packages/gatsby-plugin-image/README.md

```diff
@@ -226,11 +226,12 @@ export const query = graphql`
 
 ### Upgrading from the gatsby-image@2
 
-You can use the compat layer to make the transformation easier.
+We will be releasing a codemod to automatically update your queries and imports. In the meantime you can use the compat layer to make the transformation easier. This will be removed when we leave beta, but for now it allows you to try the component with your existing queries.
```

Comment:
```suggestion
We will be releasing a codemod to automatically update your queries and imports. In the meantime, you can use the compat layer to make the transformation easier. This will be removed when we leave beta, but for now, it allows you to try the component with your existing queries.
```